.github/workflows/deploy.yml

```diff
@@ -42,6 +42,9 @@ jobs:
           ruby-version: ${{ env.RUBY_VERSION }}
           bundler-cache: true
 
+      - name: Run bundle install
+        run: bundle install
+
```

Comment:
I don't understand why but in the 3.0 and 2.5 branches build fails with "bundle exec - not found". Ruby somehow installs correctly. I tried to copy the very same `deploy.yml` over from master but it just fails.